assets/locales/ru/translation.json

```diff
@@ -5,115 +5,117 @@
-  "deleteSelectedPresetConfirmation": "Вы уверены, что хотите удалить {{presetName}}?",
+  "noPresets": "У вас нет пресетов. Создайте во вкладке Экспертный",
+  "startServer": "Запуск сервера v{{serverVersion}}",
+  "deleteSelectedPresetConfirmation": "Вы хотите удалить {{presetName}}?",
   "savedTimestamp": "Сохранено {{when}}",
   "General": "Общее",
   "invalidType": "Неверный тип {{type}}",
-  "presetAdded": "Ваш пресет '{{presetName}}' был добавлен в список на вкладке Пресеты!",
+  "presetAdded": "Ваш пресет '{{presetName}}' добавлен! Смотрите вкладку Пресеты",
   "datetimeFormat": "MMM Do YYYY, h:mma",
   "updateDetails": "Дата выпуска: {{pubDate}}\n\nПримечания к выпуску: {{notes}}",
   "appiumIsAvailable": "Доступно Appium Desktop {{name}}",
-  "updateIsBeingDownloaded": "Обновление загружается. Вы будете уведомлены сразу по завершению",
-  "updateIsDownloaded": "Appium Desktop {{releaseName}} загружен. Необходимо перезапустить, чтобы применить обновления (примечание: это может занять несколько минут для установки и перезапуска Appium Desktop)",
+  "updateIsBeingDownloaded": "Обновление загружается. Мы уведомим о завершении",
+  "updateIsDownloaded": "Appium Desktop {{releaseName}} загружен. Для завершения обновления необходим перезапуск (это может занять несколько минут)",
   "updateDownloadFailed": "Не удалось загрузить обновление. Причина: {{message}}",
   "OK": "OK",
   "Install Now": "Установить сейчас",
   "Install Later": "Установить позже",
-  "Update Download Started": "Загрузка обновлений запущена",
+  "Update Download Started": "Запущена загрузка обновлений",
   "No update available": "Нет доступных обновлений",
   "Appium Desktop is up-to-date": "Appium Desktop обновлен",
   "Later": "Позже",
   "Update Downloaded": "Обновление загружено",
   "Could not download update": "Не удалось загрузить обновление",
-  "Inspect element": "Проверить элемент",
+  "Inspect element": "Исследовать элемент",
   "Session closed due to inactivity": "Сессия закрыта из-за неактивности",
   "Session has been terminated": "Сессия была прервана",
   "frameworkNotSupported": "Фреймворк '{{framework}}' не поддерживается",
   "stopServerFailed": "Не удалось остановить сервер: {{message}}",
-  "logsNotAvailableError": "Произошла ошибка: Журналы недоступны",
-  "findElementFailure": "Не удалось найти элемент для '{{methodName}}'. Пожалуйста, обновите страницу и повторите попытку.",
+  "logsNotAvailableError": "Ошибка: журналы недоступны",
+  "findElementFailure": "Не удалось найти элемент для '{{methodName}}'. Пожалуйста, обновите страницу и повторите попытку",
   "Could not start session": "Не удалось запустить сессию",
   "couldNotConnect": "Не удалось подключиться к серверу; вы уверены, что он запущен?",
-  "sauceCredentialsRequired": "Требуется имя пользователя и ключ доступа к Sauce!",
-  "testingbotCredentialsRequired": "TestingBot key and secret are required!",
+  "sauceCredentialsRequired": "Требуется данные для входа в Sauce!",
```

Comment:
I still think that stating explicitly about `username` and `access key` in translation is better as for saucelabs newcomers it may not be obvious.